cvat/apps/engine/migrations/0084_honeypot_support.py

```diff
@@ -39,6 +41,29 @@ def get_segment_rel_frame_set(db_segment) -> Collection[int]:
 
     return sorted(get_rel_frame(abs_frame, db_data) for abs_frame in frame_set)
 
+def cleanup_invalid_data(apps):
+    Task = apps.get_model("engine", "Task")
+    Job = apps.get_model("engine", "Job")
+
+    broken_tasks = Task.objects.annotate(
+        ground_truth_jobs_count=Count(
+            'segment__job', filter=Q(segment__job__type='ground_truth')
+        )
+    ).filter(ground_truth_jobs_count__gt=1).values_list('segment__task__id', flat=True).all()
+
+    gt_jobs = Job.objects.filter(
+        segment__task__id__in=broken_tasks
```

Comment:
```suggestion
    ).filter(ground_truth_jobs_count__gt=1)

    gt_jobs = Job.objects.filter(
        segment__task__in=broken_tasks
```

It's not necessary to manually fetch the IDs; Django will automatically update the subquery to drop all fields other than the task ID.